src/lib/eliom_content_.client.ml

```diff
@@ -396,7 +407,18 @@ module Html5 = struct
 
   module D = struct
     include Html5.D
-    include Eliom_registration_base.Html5_forms.D
+    include
+      Eliom_form.Make(struct
+        include Html5.D
+        module Svg = Eliom_content_core.Svg.D
+        let uri_of_fun = Eliom_content_core.Xml.uri_of_fun
+        let attrib_of_service s info =
+          Eliom_content_core.
+            (Html5.D.to_attrib
+               (Xml.internal_event_handler_attrib
+                  s (Xml.internal_event_handler_of_service info)))
+        let attrib_onclick = Eliom_content_core.Html5.D.a_onclick
```

Comment:
`attrib_onclick` seems to always be the same than `a_onlick`. Maybe it could be removed ?